### PR TITLE
website redesign

### PR DESCRIPTION
**Pull request by PixelatedLagg**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -1,144 +1,48 @@
...
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="index.js"></script>
+    </head>
+    <body>
+        <div class="nav">
+            <h2>Random Word API</h2>
+            <h3 onclick="endpoints();">Endpoints</h3>
+            <h3 onclick="parameters();">Parameters</h3>
+            <h3 onclick="about()">About</h3>
+            <h3 onclick="window.location.replace('https://github.com/RazorSh4rk/random-word-api');">Github</h3>
```
Comment (RazorSh4rk) [inline templates/index.html:19]: Could you just make this a link pls, you cant use the back button like this

File: templates/index.html [hunk 2]
```diff
@@ -1,144 +1,48 @@
...
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+        <link rel="stylesheet" href="index.css" type="text/css">
+        <noscript><meta http-equiv="refresh" content="0; url=error.html"/></noscript>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="index.js"></script>
+    </head>
+    <body>
+        <div class="nav">
+            <h2>Random Word API</h2>
+            <h3 onclick="endpoints();">Endpoints</h3>
```
Comment (RazorSh4rk) [inline templates/index.html:16]: If you could have some pointer that im looking at >this< page right now that would make this perfect

**Review by RazorSh4rk — CHANGES_REQUESTED** [review]
The site looks great, way better than what i could do :) Could you please do those 2 small things i mentioned?

**Comment by PixelatedLagg** [comment]
working on replacing headers with links and smooth page indication

**Pull request closed without merging**